sources.list

```diff
@@ -1,27 +1,27 @@
 # See http://help.ubuntu.com/community/UpgradeNotes for how to upgrade to
 # newer versions of the distribution.
 
-deb http://archive.ubuntu.com/ubuntu/ trusty main restricted
-# deb-src http://archive.ubuntu.com/ubuntu/ trusty restricted main multiverse universe
+deb http://archive.ubuntu.com/ubuntu/ xenial main restricted
```

Comment:
Can we use `Yakkety` instead of `Xenial` please, I know it's lifetime is lower than `Xenial` but it is a lot more awesome, powerfull and quick